CDS/js/media-atom-lib.js

```diff
@@ -6,6 +6,7 @@ var hmac = require('./hmac');
 const urlBase = process.env.url_base;
 const assetPath = '/api2/atom/:id/asset';
 const metadataPath = '/api2/atom/:id'
+const youtubePrefix = 'https://www.youtube.com/watch?v='
```

Comment:
when we have done this elsewhere, the prefix is changeable in the route; would be good to offer this option, but with this value as the default